include/iarc7_fc_comms/MspCommands.hpp

```diff
@@ -174,7 +177,6 @@ namespace FcComms
             // Jetson runs in little endian mode and the FC
             // Receives data in big endian
             // Jetson sends back 18 channels, two bytes each.
-            #pragma GCC warning "Get rid of this hardcoded number"
             for(uint32_t i = 0; i < 3; i++)
```

Comment:
Yeah I wasn't sure if you meant the 3 on this one.  Could there ever be anything other than 3 attitude numbers?  I don't think there could.